### ISSUE TITLE
1.16 - Dungeon resource

### ISSUE THREAD

**Issue by BiomeBundle** [PG85/OpenTerrainGenerator#621]
Dungeon resource is causing console errors (Failed to fetch mob spawner entity at X,Y,Z) and they are spawning cut in half or not at all.

![image](https://user-images.githubusercontent.com/23154920/117550270-558e2180-b037-11eb-9ce4-d23060bd3943.png)

![image](https://user-images.githubusercontent.com/23154920/117550274-5de65c80-b037-11eb-9777-8c1a9de26ab9.png)

![image](https://user-images.githubusercontent.com/23154920/117550279-663e9780-b037-11eb-90db-26c47024ab90.png)

**Comment by PG85** [comment]
ForgeWorldGenRegion
```
	@Override
	public void placeDungeon(Random random, int x, int y, int z)
	{
		// TODO: dungeons don't spawn due to us not using cave air in caves
		Features.MONSTER_ROOM.place(this.worldGenRegion, this.chunkGenerator, random, new BlockPos(x, y, z));
	}
```

**Comment by SXRWahrheit** [comment]
This occurs when dungeons intersect chunk borders.

**Comment by PG85** [comment]
Should be fixed for 0.0.14
